vpc/tool/container/setup_container_linux.go

```diff
@@ -32,51 +32,81 @@ var (
 	errAddressSetupTimeout = errors.New("IPv6 address setup timed out")
 )
 
-func doSetupContainer(ctx context.Context, netnsfd int, bandwidth, ceil uint64, jumbo bool, allocation types.LegacyAllocation) (netlink.Link, error) {
-	parentLink, err := getLinkByMac(allocation.MAC)
-	if err != nil {
-		return nil, err
+func getNsFd(netNS interface{}) (int, bool, error) {
```

Comment:
This is not very go-like.

Are you sure it isn't simpler to just pass 2 nsfds all the time instead of overloading the `netNS` object?